projects/plugins/boost/app/lib/critical-css/Critical_CSS_State.php

```diff
@@ -171,6 +177,9 @@ public function get_state_error() {
 	 * @param string $key Provider key.
 	 */
 	public function set_source_success( $key ) {
+		// Remove error data from previous attempts
```

Comment:
The comment explains what happens, but I have no idea why we need to do that 🙂

Reply:
I hope it's clearer with the comment.

> If this success was the result of a retry by Cloud_CSS_Cron. This provider may contain error data from the original attempt. We have to remove that first.

Reply:
Excellent. Thank you!